routes/api/hacker.js

```diff
@@ -28,6 +28,12 @@ module.exports = {
          * @apiGroup Hacker
          * @apiVersion 0.0.8
          * 
+         * @apiParam (body) {MongoID} accountId ObjectID of the respective account
+         * @apiParam (body) {String} school Name of the school the hacker goes to
+         * @apiParam (body) {String} gender Gender of the hacker
+         * @apiParam (body) {Boolean} needsBus Whether the hacker requires a bus for transportation
+         * @apiParam (body) {Object} application The hacker's application
```

Comment:
Not too big, but it would be nice to tell the user what is required inside of the application